torch/_export/serde/serialize.py

```diff
@@ -459,18 +459,18 @@ def serialize_metadata(self, node: torch.fx.Node) -> Dict[str, str]:
         if stack_trace := node.meta.get("stack_trace"):
             ret["stack_trace"] = stack_trace
 
+        st_delimiter = ";"
```

Comment:
looks like this is reused by deserialization, maybe it should be moved outside of the function